17thblog/blog.php

```diff
@@ -0,0 +1,17 @@
+<html>
+	<head>
+		<title> This is the blog part</title>
+		<link rel='stylesheet' href='style.css'/>
+	</head>
+	<body class = 'bg'>
+		<?php include 'navigationBar.php'; ?>
```

Comment:
Great job figuring this out. Even better though would be to include a file at the top of this (and every) page that defines a function called `echoNavigationBar()` or something like that and then you just call it here.

Even better still: what if you included all the boilerplate above this (html, head, title, link, etc.) in that function so you could just call a single `echoHeader()` function and leave everything up to this point on the page out.